docs/pages/features/enhanced-session-recording.mdx

```diff
@@ -4,37 +4,48 @@ description: How to record your SSH and Kubernetes sessions into the audit log.
 h1: Enhanced Session Recording
 ---
 
-Teleport [SSH and Kubernetes session recording](../architecture/nodes.mdx#session-recording)
-feature captures what is echoed to a terminal.
+Teleport enhances your session auditing and logging with powerful features including:
 
-This has inherent advantages, for example because no input is captured, Teleport
-session recordings typically do not contain passwords that were entered into a terminal.
+1. Video session recording and playback
```

Comment:
I'd do another round of edits to make it factually correct:

* Enhanced session recording is using BPF and prevents obfuscation techniques. Requires BPF and newer kernels.
* Session recording (the usual one) is capturing Pseudo-Terminal output and is on by default.
So usual session recording is not a video but a raw PTY output capture.

Reply:
I don't believe that's factually incorrect per see (?), but it's (much) less comprehensive than it could be ... so this needs a third point to better describe the default session recording. Say:

1. **Session Recording (Default):**  ...
2. **Session Playback:** ...
3. **Enhanced Session Recording**: ...

Is that more accurate?

Hmmm ... I think the previous intro makes it seem that there are exactly two options here when there are several other settings that can be configured or enabled (for example events/triggers).